### PR TITLE
CI: Release update and screenshot gathering automation

### PR DESCRIPTION
**Pull request by kz6fittycent**
- Modified script for getting latest release from GH API
- Minor cleanup of some old comments in yml files
- Update snapraft.yaml to `version` instead of `adopt-info`

I searched for anything that may be lacking, but let me know if/what I missed.

### CODE REVIEW

**Review by merlijn-sebrechts — CHANGES_REQUESTED** [review]
I see some of these actions are still using the old in-repo ones. Can you take a look at https://github.com/snapcrafters/signal-desktop/tree/candidate/.github/workflows and update accordingly?

I apologize for this churn. The current ones in signal-desktop should be the final ones, I promise!

**Comment by kz6fittycent** [comment]
> I see some of these actions are still using the old in-repo ones. Can you take a look at https://github.com/snapcrafters/signal-desktop/tree/candidate/.github/workflows and update accordingly?
> 
> I apologize for this churn. The current ones in signal-desktop should be the final ones, I promise!

No problem! I was fully expecting to redo something because of my own errors anyway. Give me a couple of days to get to it.

File: .github/vmctl [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
```
Comment (merlijn-sebrechts) [inline .github/vmctl]: You can remove this entire file. The new workflow downloads the latest version automatically

**Review by merlijn-sebrechts — CHANGES_REQUESTED** [review]
A small change. Otherwise, it looks good!

**Comment by merlijn-sebrechts** [comment]
I've added the tokens, so this should work as soon as we merge it!

File: .github/vmctl (hunk 1 above)
Comment (kz6fittycent) [reply .github/vmctl]: stand by

**Review by merlijn-sebrechts — APPROVED** [review]
Guddd!

**Pull request merged**